### PR TITLE
Create FOXE-FOXEERF405_24A.config

### PR DESCRIPTION
**Pull request by Johnsnow-foxeer**
Please add Foxeer F405 target

### CODE REVIEW

File: configs/default/FOXE-FOXEERF405_24A.config [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+board_name FOXEERF405_24A
```
Comment (haslinghuis) [inline configs/default/FOXE-FOXEERF405_24A.config:14]: I suggest using FOXEERF405_AIO to be consistent with

image

File: configs/default/FOXE-FOXEERF405_24A.config [hunk 2]
```diff
@@ -0,0 +1,131 @@
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO
```
Comment (haslinghuis) [inline configs/default/FOXE-FOXEERF405_24A.config:5]: Already defined on line 3

Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF405_24A.config:5]: removed, thank you

File: configs/default/FOXE-FOXEERF405_24A.config (hunk 1 above)
Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF405_24A.config:13]: OK, please delete this PR, I will rename the fw name

**Pull request closed without merging**